scripts/build-renode.sh

```diff
@@ -58,17 +58,48 @@ if [ ! -z "$ETH_BASE_ADDRESS" ]; then
 		# Build/copy the image into the TFTP directory.
 		make tftp
 
-		TAP_INTERFACE=tap0
+		RENODE_CONFIG="--configure-network tap0"
+		;;
+
+	internal)
+		echo "Using the Renode internal TFTP server for netbooting"
+
+		if [ "$FIRMWARE" = "linux" ] && [ "$CPU" = "vexriscv" ]; then
+			RENODE_CONFIG="--tftp-binary \"$TARGET_BUILD_DIR/software/linux/firmware.bin:Image\"
+				--tftp-binary \"$TARGET_BUILD_DIR/software/linux/boot.json\"
+				--tftp-binary \"$TARGET_BUILD_DIR/emulator/emulator.bin\"
+				--tftp-binary \"$TARGET_BUILD_DIR/software/linux/rv32.dtb\"
+				--tftp-binary \"$TARGET_BUILD_DIR/software/linux/riscv32-rootfs.cpio:rootfs.cpio\""
+		else
+			RENODE_CONFIG="--tftp-binary \"$TARGET_BUILD_DIR/software/$FIRMWARE/firmware.bin:boot.bin\""
+		fi
+
+		RENODE_CONFIG="$RENODE_CONFIG
+			--tftp-server-ip \"192.168.100.100\"
+			--tftp-server-port 6069"
 		;;
 
 	none)
-		echo "Renode networking disabled..."
+    	        # this will be handled later
```

Comment:
Currently this sounds like a "TODO: Add handling here" rather than "This part of the case statement is handled separately at line 90." 

```suggestion
    	        # This case is handled a bit further below.
```

Reply:
You are right, I rephrased it as suggested.